### ISSUE TITLE
Release Twisted 16.5.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8816]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8816|
|Type|task|
|Created|2016-09-14 17:51:31Z|
|Branch|https://github.com/twisted/twisted/tree/release-16.5.0-8816|

<details><summary>Searchable metadata</summary>

```
trac-id__8816 8816
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__release_16_5_0_8816 release-16.5.0-8816
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__None None
time__1473875491116201 1473875491116201
changetime__1477723065853027 1477723065853027
version__None None
owner__hawkowl hawkowl
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Builds are passing except for some minor doc building blahs. Pls review.

(No PR, this one will be a manual merge because git is stupid and won't remove the cherrypicked topfiles)

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_4"></a>|@Lukasa set owner to @hawkowl|
|-|-|

Looks good to me. Release when ready.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 2bc748aec48d4c734a5cf96d408c734611e05972
```
#!CommitTicketReference repository="" revision="2bc748aec48d4c734a5cf96d408c734611e05972"
Merge branch 'release-16.5.0-8816': Release Twisted 16.5

Author: hawkowl
Reviewer: lukasa
Fixes: #8816
```
